api/pkg/apis/v1alpha1/utils/parser.go

```diff
@@ -559,6 +559,26 @@ func (n *FunctionNode) Eval(context utils.EvaluationContext) (interface{}, error
 			return property, nil
 		}
 		return nil, v1alpha2.NewCOAError(nil, fmt.Sprintf("$output() expects 2 argument, found %d", len(n.Args)), v1alpha2.BadConfig)
+	case "trigger":
```

Comment:
@iwangjintian @linyguo , this PR is related to parser. Please help to review as well.